hudsonrock/plugin.go

```diff
@@ -0,0 +1,26 @@
+package hudsonrock
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+const pluginName = "steampipe-plugin-hudsonrock"
+
+// Plugin returns the Hudson Rock plugin definition.
+func Plugin(ctx context.Context) *plugin.Plugin {
+	return &plugin.Plugin{
+		Name:             pluginName,
+		DefaultTransform: transform.FromGo().NullIfEmptySlice(),
+		TableMap: map[string]*plugin.Table{
+			"hudsonrock_username_search": tableHudsonrockUsernameSearch(ctx),
+			"hudsonrock_email_search":    tableHudsonrockEmailSearch(ctx),
+			"hudsonrock_domain_search":   tableHudsonrockDomainSearch(ctx),
+			"hudsonrock_ip_search":       tableHudsonrockIpSearch(ctx),
+			"hudsonrock_urls_by_domain":  tableHudsonrockUrlsByDomain(ctx),
+		},
+	}
```

Comment:
- Should you include the default `DefaultIgnoreConfig`? Ref: https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/plugin.go#L39
- `ConnectionConfigSchema` is missien here. Ref: https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/plugin.go#L48C3-L48C25